docs/FAQ.md

```diff
@@ -515,6 +515,27 @@ by `jj rebase` to rebase the changes in a commit. It's used in `jj log` to
 indicate which commits are empty. It's used in the `files()` revset function
 (and by `jj log <path>`) to find commits that modify a certain path. And so on.
 
+## How do I revert a merge commit? `jj revert <merge>` does nothing
+
+Instead use `jj restore --from <first parent>` to revert the changes merged
+from the second parent.
+
+Example:
+```text
+@
+|
+C
+| \
+B D
+|/
+A
+```
+To revert the merge in `C`, `jj restore --from B` and then describe the message
```

Comment:
I think you should add `jj new C` before `jj restore`.

I previously wrote this above in an about to be deleted comment. I'm more sure about needing it here than above.

Reply:
done